src/grammar.rs

```diff
@@ -859,8 +866,9 @@ impl Grammar {
                 stype,
                 &next_node.with_reduce_vec(reduce_vec),
             )
-            .expect("Double conflict!");
+            .map_err(|_| Diagnostic::GrammarCantBuild)?; // Double conflict
```

Comment:
```suggestion
            .map_err(|_| Diagnostic::GrammarCantBuild("Double conflict"))?;
```
It's a shame to lose these comments in the output. If it stores a `&'static str` or `Cow<'static, str>` then you don't have to go to the trouble of creating an enum for these messages but you can still display them

Reply:
Yes, good suggestion. I'll do that.